ceph/ceph_admin/bootstrap.py

```diff
@@ -54,15 +93,15 @@ def bootstrap(self: CephAdmProtocol, config: Dict) -> None:
             cmd += f" --image {self.config['container_image']}"
 
         cmd += " bootstrap"
-        custom_image_args = (
-            " --registry-url registry.redhat.io"
-            " --registry-username {user}"
-            " --registry-password {password}"
-        )
-        cmd += custom_image_args.format(
-            user=cdn_cred.get("username"),
-            password=cdn_cred.get("password"),
-        )
+
+        # Construct registry credentials as string or json.
+        registry_url = args.pop("registry-url", None)
```

Comment:
This changes the default behavior. If `registry_url` or `registry_json` is required, could you update `cephadm_deploy.yaml` also.

Reply:
Done